File: plugins/textmate/tests/org/jetbrains/plugins/textmate/language/syntax/lexer/HtmlVscLexerTest.java
package org.jetbrains.plugins.textmate.language.syntax.lexer;

import org.jetbrains.plugins.textmate.TestUtil;

public class HtmlVscLexerTest extends LexerTestCase {
  @Override
  protected String getBundleName() {
    return TestUtil.HTML_VSC;
  }

  @Override
  protected String getTestDirRelativePath() {
    return "html_vsc";
  }
}

File: plugins/textmate/tests/org/jetbrains/plugins/textmate/language/syntax/lexer/HtmlVscLexerTest.kt
package org.jetbrains.plugins.textmate.language.syntax.lexer;

import org.jetbrains.plugins.textmate.TestUtil;

import java.util.Collections;
import java.util.List;

public class HtmlVscLexerTest extends LexerTestCase {
  @Override
  protected String getBundleName() {
    return TestUtil.HTML_VSC;
  }

  @Override
  protected List<String> getExtraBundleNames() {
    return Collections.singletonList(TestUtil.CSS_VSC);
  }

  @Override
  protected String getTestDirRelativePath() {
    return "html_vsc";
  }
}
